### PR TITLE
RPC calls: Increase endpoints calls

### PR DESCRIPTION
**Pull request by stunko**

### CODE REVIEW

File: integration/tests/basic/helpers/json_rpc_requester.py [hunk 1]
```diff
@@ -1,45 +1,42 @@
...
+            payloads = dataclasses.asdict(payloads)
+        with allure.step(f"Request params: {payloads}"):
+            response = self._session.post(self._url, json=payloads)
+            return self._deserialize_response(response, model=model)
+
+    @staticmethod
+    def _deserialize_response(
+        response: Response, model: tp.Any = None
+    ) -> tp.Union[JsonRpcResponse, JsonRpcErrorResponse]:
+        json_doc = response.json()
```
Comment (gigimon) [inline integration/tests/basic/helpers/json_rpc_requester.py]: here can be not json, catch json deserealize error

**Review by gigimon — APPROVED** [review]

File: integration/tests/basic/helpers/assert_message.py [hunk 2]
```diff
@@ -8,3 +8,7 @@ class AssertMessage(Enum):
     WRONG_AMOUNT = "Wrong amount returned"
     CONTAINS_ERROR = "Contains the error data"
     DOES_NOT_CONTAIN_RESULT = "Does not contain the result data"
+    IVALID_ARG_1_GIVEN = "takes 0 positional arguments but 1 was given"
```
Comment (apetrovskiy) [inline integration/tests/basic/helpers/assert_message.py]: Should be INVALID_...

Comment (apetrovskiy) [inline integration/tests/basic/helpers/assert_message.py:11]: Should be INVALID_...

Comment (gigimon) [inline integration/tests/basic/helpers/assert_message.py]: For this message we must create bug! This error message raise Traceback in logs!!!!

File: integration/tests/basic/test_rpc_calls.py [hunk 3]
```diff
@@ -25,166 +30,358 @@
...
+        payloads = RpcRequestFactory.get_block_by_hash(params=params)
+
+        actual_result = self.json_rpc_client.do_call(payloads, request_models.BlockResponse)
+
+        assert actual_result.id == payloads.id, AssertMessage.WRONG_ID.value
+        assert self.assert_no_error_object(actual_result), AssertMessage.CONTAINS_ERROR
+        assert self.assert_result_object(actual_result), AssertMessage.DOES_NOT_CONTAIN_RESULT
+
+    @pytest.mark.parametrize("quantity_tag,full_trx", TAGS_TEST_DATA)
+    def test_eth_get_blockn_by_umber_via_tags(self, quantity_tag: tp.Union[int, Tag], full_trx: bool):
```
Comment (gigimon) [inline integration/tests/basic/test_rpc_calls.py]: check test name

File: integration/tests/basic/test_rpc_calls.py [hunk 4]
```diff
@@ -25,166 +30,358 @@
...
+            assert len(response.result) >= 3, f"Invalid response result: {response.result}"
+
+    @pytest.mark.parametrize("params, raises", [(None, False), ("param", True)])
+    def test_eth_hash_rate(self, params, raises):
+        """Verify implemented rpc calls work eth_hashrate"""
+        response = self.assert_rpc_response(method="eth_hashrate", params=params, raises=raises)
+        if not raises:
+            assert self.is_hex(response.result), f"Invalid response result: {response.result}"
+
+    def test_check_unsupported_methods(self):
```
Comment (gigimon) [inline integration/tests/basic/test_rpc_calls.py]: use parametrize and @pytest.xfail for this. It's more clear for pytest

**Review by gigimon — APPROVED** [review]

**Review by gigimon — APPROVED** [review]

**Pull request merged**